sdk/core/azure-core/test/ut/datetime.cpp

```diff
@@ -457,3 +455,17 @@ TEST(DateTime, ParseTimeInvalid2)
     EXPECT_EQ(dt.ToInterval(), 0);
   }
 }
+
+TEST(DateTime, ParseDatesBefore1900)
+{
+  TestDateTimeRoundtrip("1899-01-01T00:00:00Z");
+  auto dt1 = DateTime::FromString("1899-01-01T00:00:00Z", DateTime::DateFormat::Iso8601);
+  auto dt2 = DateTime::FromString("Sun, 1 Jan 1899 00:00:00 GMT", DateTime::DateFormat::Rfc1123);
+  EXPECT_EQ(dt1.ToInterval(), dt2.ToInterval());
+
+  TestDateTimeRoundtrip("1601-01-01T00:00:00Z");
```

Comment:
Please also add a test for a date prior to 1600 to capture edge case behavior.

Reply:
"time zone underflow" and "year too small" above are updated to test that.